pkg/test/metrics/metric.go

```diff
@@ -12,6 +12,14 @@ func AssertMetricsCounterEquals(t *testing.T, expected int, c prometheus.Counter
 	assert.InDelta(t, float64(expected), promtestutil.ToFloat64(c), 0.01)
 }
 
+func AssertCounterEqualsInt(t *testing.T, expected int, c prometheus.Counter) {
+	assert.Equal(t, expected, int(promtestutil.ToFloat64(c)))
```

Comment:
is it actually required to convert to float64 and than to int? I believe so, but just curious.

Reply:
yes, if you want to compare to an `int` in the `assert.Equal` func.